tx_clients/clients/line_client.py

```diff
@@ -1,61 +0,0 @@
-from txconnpool.pool import PooledClientFactory, Pool
-
-from tx_clients.protocols import LineProtocol
-from tx_clients.utils import Retry
-from tx_clients.exceptions import TimeoutError
-
-
-class PooledLineProtocol(LineProtocol):
-    """
-    A LineProtocol that will notify a connectionPool that it is ready
-    to accept requests.
-    """
-
-    def connectionMade(self):
-        """
-        Notify our factory that we're ready to accept connections.
-        """
-        LineProtocol.connectionMade(self)
-
-        self.factory.connectionPool.clientFree(self)
-
-        if self.factory.deferred is not None:
-            self.factory.deferred.callback(self)
-            self.factory.deferred = None
-
-
-class PooledLineClientFactory(PooledClientFactory):
-    protocol = PooledLineProtocol
-
-
-class LineClientPool(Pool):
```

Comment:
This is actually still used in titan and titan master is borken now.

https://github.com/pantheon-systems/titan-mt/blob/master/yggdrasil/connection.py#L5


@Hornswoggles, can you pls update titan, if it should use other tx-client API?